pyvelox/test/test_vector.py

```diff
@@ -254,3 +256,22 @@ def test_append(self):
 
         with self.assertRaises(TypeError):
             ints2.append(strs2)
+
+
+class TestVeloxVectorSaver(unittest.TestCase):
+    def setUp(self):
+        # create a temporary directory
+        self.test_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        # remove the temporary directory
+        shutil.rmtree(self.test_dir)
+    
+    def test_serde_vector(self):
+        vec = pv.from_list([1, 2, 3])
```

Comment:
Can you add a test for different encodings of vectors : 
1 - Constant 
2 - Dictionary

Also can you add string and float to this ?